test/parallel/test-http2-window-size.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+// This test ensures that servers are able to send data independent of window
+// size.
+
```

Comment:
There is a chance, on some smaller / iot platforms this test could be problematic because of the large buffer allocations. May be a  good idea to include a TODO note in here to test on those machines... and if it ends up being too much of a burden, we might want to move this test out of parallel.